amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -1968,6 +1968,39 @@ extern "C"
 	 */
 	SHARED_PUBLIC vx_node VX_API_CALL vxExtRppTensorAddTensor(vx_graph graph, vx_tensor pSrc1, vx_tensor pSrc2, vx_tensor pDst, vx_tensor pSrcDims, vx_tensor pDstDims, vx_uint32 nbatchSize);
 
+	/*! \brief [Graph] Performs leading and trailing silence detection to the input tensor.
+	* \ingroup group_amd_rpp
+	* \param [in] graph The handle to the graph.
+	* \param [in] pSrc The input tensor in <tt>\ref VX_TYPE_FLOAT32</tt> format data.
+	* \param [in] pSrcRoi The input tensor of batch size in <tt>unsigned int<tt> containing the roi values for the input in xywh/ltrb format.
+	* \param [out] pDst1 The output tensor (begin) in <tt>\ref VX_TYPE_INT32</tt> format data.
+	* \param [out] pDst2 The output tensor (length) in <tt>\ref VX_TYPE_INT32</tt> format data.
+	* \param [in] cutOffDB The input array in <tt>\ref VX_TYPE_FLOAT32</tt> format containing the threshold, in dB, below which the signal is considered silent.
+	* \param [in] referencePower The input array in <tt>\ref VX_TYPE_FLOAT32</tt> format containing the reference power.
+	* \param [in] windowLength The input array in <tt>\ref VX_TYPE_INT32</tt> format containing the size of the sliding window.
+	* \param [in] resetInterval The input array in <tt>\ref VX_TYPE_INT32</tt> format containing the frequency at which the moving mean average is recalculated to mitigate precision loss.
+	* \return A node reference <tt>\ref vx_node</tt>. Any possible errors preventing a successful creation should be checked using <tt>\ref vxGetStatus</tt>.
+	*/
+	SHARED_PUBLIC vx_node VX_API_CALL vxExtRppNonSilentRegion(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcRoi, vx_tensor pDst1, vx_tensor pDst2, vx_scalar cutOffDB, vx_scalar referencePower, vx_scalar windowLength, vx_scalar resetInterval);
+
+	//TODO: Change the explanations
+	/*! \brief [Graph] Slice's the input tensor
+	* \ingroup group_amd_rpp
+	* \param [in] graph The handle to the graph.
+	* \param [in] pSrc The input tensor in <tt>\ref VX_TYPE_UINT8</tt> or <tt>\ref VX_TYPE_FLOAT32</tt> or <tt>\ref VX_TYPE_FLOAT16</tt> or <tt>\ref VX_TYPE_INT8</tt> format data.
+	* \param [in] pSrcRoi The input tensor of batch size in <tt>unsigned int<tt> containing the roi values for the input in xywh/ltrb format.
+	* \param [out] pDst1 The output tensor (begin) in <tt>\ref VX_TYPE_UINT8</tt> or <tt>\ref VX_TYPE_FLOAT32</tt> or <tt>\ref VX_TYPE_FLOAT16</tt> or <tt>\ref VX_TYPE_INT8</tt> format data.
```

Comment:
It should not be pDst1 and pDst here
it should be anchor and shape and the datatype is only int

Please check and change